package.json

```diff
@@ -68,14 +68,30 @@
     "url": "http://modernizr.com/"
   },
   "contributors": [
-    { "name": "Faruk Ates" },
-    { "name": "Paul Irish" },
-    { "name": "Alex Sexton" },
-    { "name": "Ryan Seddon" },
-    { "name": "Alexander Farkas" },
-    { "name": "Patrick Kettner" },
-    { "name": "Stu Cox" },
-    { "name": "Richard Herrera" }
+    {
+      "name": "Faruk Ates"
```

Comment:
That's how npm saves it. I normally manually reformat it back because it bugs the shit out of me